client/server/middleware/logger.js

```diff
@@ -49,7 +50,9 @@ export default () => {
 	const version = process.env.COMMIT_SHA;
 
 	return ( req, res, next ) => {
-		req.logger = logger;
+		req.logger = logger.child( {
+			reqId: uuidv4(),
+		} );
```

Comment:
The `child` method has a second, not very well documented, boolean param called `simple`. In cases where the child logger only adds fields and nothing else, it promises to create the children faster.

https://github.com/trentm/node-bunyan/blob/master/lib/bunyan.js#L681-L684

Reply:
I saw that, but I rather not used an undocumented method because they may remove it at any time and not even add it to the changelog notes. We can revisit if we have any suspicion that creating a child logger is slowing things down.